cis/plugins/validation/mozilliansorg_publisher_plugin.py

```diff
@@ -57,6 +64,14 @@ def run(publisher, user, profile_json):
     old_groups = user.get('groups')
     new_groups = profile_json.get('groups')
 
+    for profile_idp in whitelist_idp_with_enforced_mfa:
+        if profile_json.get('user_id').startswith(profile_idp):
+            if new_groups and len(new_groups) != 0:
```

Comment:
probably you don't need the `len(new_groups)`. if the first check, returns true, that means that the list is not empty and we have groups.